04_environment/design.tex

```diff
@@ -0,0 +1,289 @@
+
+Furthermore, the disaster was designed to comprise of an epicentre (eye of the storm), which is limited within the archipelago bounds, it is represented using Cartesian coordinates (EpiX, EpiY). Therefore, all islands or a subset of islands may be affected due to their proximity to the disaster epicentre. The island’s proximity to the disaster epicentre is directly proportional to the island’s damage. In other words, the island's available resources will be diminished based on the disaster magnitude and its location with respect to each island’s fixed location.
+
+The following example can briefly explain the dilemma:
+
+The closest an island from the eye of the storm, the larger the severity on that island. Therefore, the more resources are depleted. Let’s say the disaster epicentre hits around Island 1 as shown in Figure~\ref{fig:Disaster eye of the storm severity} and that the desired common pool threshold has not been met. Island 1 would be experiencing severe damages , and Island 2 and 6 some damages and the other islands low to no damage. Therefore, Island 1, 2 and 6 have the risk of not surviving if they are lacking resources, whereas Island 3, 4, and 5 would not be affected as much by the disaster.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/images/Disaster eye of the storm severity.PNG}
+    \caption{Disaster Epicenter effects}
+    \label{fig:Disaster eye of the storm severity}
+\end{figure}
+
+\newpage
+\subsection{Future work}
+
+There are multiple future work ideas that would be developed in the design aspect such as; a deterministic disaster which would be  designed as a straight line that accumulates during the days, once the threshold was met then the disaster would occur. Another deterministic disaster idea would be to link the disaster’s magnitude to time. Thus, agents would have to learn from past disaster occurrences, creating a memory. Agents would start learning from past disasters and would be able to forecast the next one. Therefore, the agents would be able to forecast that if a disaster did not surface for a long period of time, then the magnitude of the next disaster will be much larger than the previous one.
+
+Another idea is that agents can invest into forecasting. Therefore, agents will have access to a history of past disasters, which would be utilised to learn and gain knowledge in order to predict the upcoming disasters.
+
+Thus, the more resources invested by the agents in forecasting, the more disaster history data will be provided. This would enable the agents, to apply machine learning techniques to such data, allowing them to acquire more accurate and precise predictions of future events and develop better risk assessments. Therefore, agents will acquire knowledge, when investing in any of their resources.
```

Comment:
no need here for additional paragraphs either - new paragraphs are for new ideas but you continue with the thus which is a bit confusing - joining them together is a quick fix :)